lib/index.coffee

```diff
@@ -20,9 +22,27 @@ module.exports =
       loadModule()
       packageSync.sync()
 
+    atom.packages.onDidActivateInitialPackages ->
+      @didActivateInitialPackages = true
```

Comment:
You're right, I had been trying to account for changing the setting after launch but this isn't necessary. Something that I'm not sure about is a user enabling this setting and expecting it to work after installing the package, but before relaunching the program and giving the handlers a chance to be registered `onDidActivateInitialPackages`.